algorithms/Python/Tree/binary_search_tree.py

```diff
@@ -0,0 +1,238 @@
+"""
+Binary Tree Data Structure
+    Data is stored in hierarchical form where a parent node can have at most 2 child nodes
+                A
+             ___|___
+            B       C
+         ___|___
+        D       E
+      /  \       \
```

Comment:
the python test if failing due to a presence of a stray backslash inside the string literal. 

There are 2 possible fixes that I can think of right now 

1.  Upgrading the ASCII art of the tree to not contain backslashes 
2. Not using string literal as comments and use conventional comment (using `#`) instead

Reply:
Is it still failing?
Code with backslashes and string literals is also removed.